packit/sync.py

```diff
@@ -29,7 +29,7 @@ def check_subpath(subpath: Path, path: Path) -> Path:
     Raises:
         PackitException, if 'subpath' is not a subpath of 'path'.
     """
-    if not subpath.resolve().is_relative_to(path.resolve()):
+    if not is_relative_to(subpath.resolve(), path.resolve()):
```

Comment:
since it's being used only here and we now know how it's implemented, I'd do the try/except dance here:
```python
is_relative = True
# we cannot use is_relative_to b/c it's not available on el8
try:
    subpath.resolve().relative_to(path.resolve())
except ValueError:
    is_relative = False
```

what do others think?

Reply:
makes sense. I though that it could be used more often.

But this is just one place, so fixed by your suggested way.

Reply:
I was just typing the same comment when Honza's new commit arrived :)